includes/init.php

```diff
@@ -347,9 +363,17 @@ function print_trailer( $include_nav_links = true, $closeDb = true,
   }
 
   return $ret . '
-<!-- ' . $GLOBALS['PROGRAM_NAME'] . '     ' . $GLOBALS['PROGRAM_URL'] . ' -->
-' // Adds an easy link to validate the pages.
-  . ( $DEMO_MODE == 'Y' ? '
+<!-- ' . $GLOBALS['PROGRAM_NAME'] . '     ' . $GLOBALS['PROGRAM_URL'] . ' -->' .
+    ( $ALLOW_HTML_DESCRIPTION !== 'Y' ? '' :
+    /* Your choices here are "basic", "standard" or "full". */ '
+    <script src="//cdn.ckeditor.com/4.6.0/basic/ckeditor.js"></script>
```

Comment:
I'm okay with loading stuff remotely as an option.  But users need to have the option to run 100% locally.  Some environments may not have access to the remote sites.  And, some people prefer to not include javascript loaded from remote servers (depending on the site).